### ISSUE TITLE
[FEATURE] Echo manifest names for `wash app` commands

### ISSUE THREAD

**Issue by thomastaylor312** [wasmCloud/wadm#175]
**Is your feature request related to a problem? Please describe.**

When I am using some of the example manifests, I don't necessarily know what the manifest's name is.

**Describe the solution you'd like**

I would really like to have the output of the `wash app` commands output the manifest name. Examples below:

```shell
$ wash app put wadm.yaml

Successfully put manifest kvcounter v0.1.0
$ wash app deploy kvcounter

Successfully deployed kvcounter v0.1.0
```

**Comment by ahmedtadde** [comment]
@brooksmtownsend  
I was digging into the related code and according to [this](https://github.com/wasmCloud/wash/blob/bb0197c1c771e7bf25d44996296687a4d02ee062/src/main.rs#L318C21-L318C21), [this](https://github.com/wasmCloud/wadm/blob/942a811079d15b59ea5c4a760748d683baac9772/src/server/handlers.rs#L97), and [this](https://github.com/wasmCloud/wash/blob/bb0197c1c771e7bf25d44996296687a4d02ee062/src/app/mod.rs#L299), the output from the commands should already include the manifest name when the user specifies the `json` format for outputs. So, it sounds like this issue is mainly for when the output format is set to `text`. Is it then the case that [this] (https://github.com/wasmCloud/wash/blob/bb0197c1c771e7bf25d44996296687a4d02ee062/src/main.rs#L321C17-L321C27) should be modified to append the manifest name + version from `out.map` to the `out.text`?

**Comment by brooksmtownsend** [comment]
@ahmedtadde That looks right to me actually, looks like `PutModelResponse` includes the name of the manifest but `DeployModelResponse` doesn't. So `wash` should include the name on put, and wadm needs to include the name on deploy

**Comment by connorsmith256** [comment]
Resolved by #178
